netbox/templates/dcim/device.html

```diff
@@ -247,10 +247,15 @@ <h5 class="card-header">
                                             <td style="padding-left: 20px">Leg {{ leg.name }}</td>
                                             <td>{{ leg.outlet_count }}</td>
                                             <td>{{ leg.allocated }}</td>
-                                            <td>{{ powerfeed.available_power|divide:3 }}VA</td>
-                                            {% with phase_available=powerfeed.available_power|divide:3 %}
-                                                <td>{% utilization_graph leg.allocated|percentage:phase_available %}</td>
-                                            {% endwith %}
+                                            {% if powerfeed.available_power %}
+                                                <td>{{ powerfeed.available_power|divide:3 }}VA</td>
+                                                {% with phase_available=powerfeed.available_power|divide:3 %}
```

Comment:
We can avoid doing the division twice:
```suggestion
                                                {% with phase_available=powerfeed.available_power|divide:3 %}
                                                    <td>{{ phase_available }}VA</td>
```